tests/Image/ImageTest.php

```diff
@@ -167,4 +167,122 @@ public function testCrop100x100GIF()
         $this->assertEquals('GIF', $image->getImageFormat());
         \unlink($target);
     }
+
+    public function testBorder5Red()
+    {
+        $image = new Image(\file_get_contents(__DIR__ . '/../resources/disk-a/kitten-1.jpg'));
+        $target = __DIR__.'/border_5_red.jpg';
+        $original = __DIR__.'/../resources/resize/border_5_red.jpg';
+
+        $image->setBorder(5, "#ff0000");
+
+        $image->save($target, 'jpg', 100);
+
+        $this->assertEquals(\is_readable($target), true);
+        $this->assertEquals(\mime_content_type($target), \mime_content_type($original));
+        $this->assertNotEmpty(\md5(\file_get_contents($target)));
+
+        $image = new \Imagick($target);
+        $this->assertEquals('JPEG', $image->getImageFormat());
+        \unlink($target);
+    }
+
+    public function testRotate45()
```

Comment:
For this you could take a non square image, rotate it and check the resolution afterwerds 👍🏻